packages/addons/service/openvfd-service/package.mk

```diff
@@ -0,0 +1,28 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2018-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="openvfd-service"
+PKG_VERSION="99b2ab997623ccc7ee6914b1b832514e36a65eda"
+PKG_SHA256="23213823d1a602217c09998ec49409ca17113c2a318541cf53e9333d8e6a169b"
+PKG_REV="100"
+PKG_ARCH="arm aarch64"
+PKG_LICENSE="GPL2"
+PKG_SITE="https://github.com/arthur-liberman/service.openvfd"
+PKG_URL="https://github.com/arthur-liberman/service.openvfd/archive/$PKG_VERSION.tar.gz"
+PKG_DEPENDS_TARGET="toolchain"
+PKG_SECTION="service"
+PKG_SHORTDESC="OpenVFD: Service for VFD displays"
+PKG_LONGDESC="OpenVFD: Service for controlling Ethernet and WiFi status, Time, Date and Playback indicators via the Linux OpenVFD driver"
+PKG_TOOLCHAIN="manual"
+
+PKG_IS_ADDON="yes"
+PKG_ADDON_NAME="OpenVFD"
+PKG_ADDON_TYPE="xbmc.service"
+PKG_ADDON_PROJECTS="Allwinner Amlogic Rockchip"
+
+addon() {
+  mkdir -p $ADDON_BUILD/$PKG_ADDON_ID
+    cp -PR $PKG_BUILD/* $ADDON_BUILD/$PKG_ADDON_ID
+    cp $PKG_DIR/changelog.txt $ADDON_BUILD/$PKG_ADDON_ID
+    cp $PKG_DIR/icon/icon.png $ADDON_BUILD/$PKG_ADDON_ID
```

Comment:
Aren't changelog and icon are copied automatically?

Reply:
@vpeter4 it picks up the changelog and addon.xml from the source package, thats wrong
the "workaround" is also incomplete - this is overall not necessary if the addon.xml, changelog and icon is removed from source package
@chewitt  http://ix.io/1y4m fixed it fo me